docs/upctl_network_create.md

```diff
@@ -0,0 +1,45 @@
+## upctl network create
+
+Create a network
+
+```
+upctl network create [flags]
+```
+
+### Options
+
+```
+      --name string              Names the network.
+      --zone string              The zone in which the network is configured.
+      --router string            Add this network to an existing router.
+      --ip-network stringArray   A network interface for the server, multiple can be declared.
+                                 
+                                  Fields: 
+                                   address: string 
+                                   family: string 
+                                   gateway: string 
+                                   dhcp: true/false 
+                                   dhcp-default-route: true/false 
+                                   dhcp-dns: array of strings 
+                                 Usage: 
+                                 	--ip-network 'address=94.23.112.143,"dhcp-dns=<value1>,<value2>",gateway=<gateway>,dhcp=true' 
```

Comment:
Are these generated from the examples done earlier? Would be nice to get these wrapped in the 'console output quotes' (backquotes?) in the markdown output if it's possinle.